package.json

```diff
@@ -13,6 +13,7 @@
     "@radix-ui/themes": "^3.1.4",
     "react": "^18.3.1",
     "react-dom": "^18.3.1",
+    "react-icons": "^5.3.0",
```

Comment:
agora que adicionou o react icons, vamos tentar manter o uso dele ok, sem mais imports para lib de icones, apenas se for svg ficar na pasta assets

Reply:
entendido.